search-app/test/cmr/search/test/site/routes.clj

```diff
@@ -7,39 +7,59 @@
                      {:public-conf {:protocol "https"
                                     :relative-root-url "/search"}}))
 
+(def base-url "https://cmr.example.com/search")
+
 (defn- substring?
   [test-value string]
   (.contains string test-value))
 
```

Comment:
Need to add more tests; since this was created, docs URLs and associated redirects have been added.